modules/tensor_mechanics/src/userobjects/NodalPatchRecoveryBase.C

```diff
@@ -62,8 +62,8 @@ NodalPatchRecoveryBase::nodalPatchRecovery(const Point & x,
   RealEigenVector b = RealEigenVector::Zero(_q);
   for (auto elem_id : elem_ids)
   {
-    A += _Ae.at(elem_id);
-    b += _be.at(elem_id);
+    A += libmesh_map_find(_Ae, elem_id);
```

Comment:
What's the advantage of libmesh map find over at?

Reply:
It gives you a backtrace if it fails. It's also been requested in PR reviews.

Reply:
Interesting... Weren't we catching this sort of exception and emitting backtrace before?

Reply:
The simulation did stop but I didn't get any backtrace, just that the key was invalid.